File: LinerLayout/app/src/main/java/com/example/linerlayout/MainActivity.java
package com.example.linerlayout;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.Spinner;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void onSendClick(View view) {
        TextView name = findViewById(R.id.textView);
        TextView group = findViewById(R.id.textView2);
        TextView message = findViewById(R.id.textView3);
        Spinner names = findViewById(R.id.spinner1);
        Spinner gpoups = findViewById(R.id.spinner2);
        EditText editText = findViewById(R.id.editText);

        message.setText(editText.getText().toString());
        editText.setText("");
        name.setText(names.getSelectedItem().toString());
        group.setText(gpoups.getSelectedItem().toString());

    }
}

File: BuildAndResources/app/src/main/java/com/example/buildandresources/MainActivity.kt
package com.example.buildandresources

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val textView = findViewById<TextView>(R.id.textView)
        textView.text = """
            buildType=${BuildConfig.BUILD_TYPE}
            flavor=${BuildConfig.FLAVOR}
            versionCode=${BuildConfig.VERSION_CODE}
            versionName=${BuildConfig.VERSION_NAME}
            applicationId=${BuildConfig.APPLICATION_ID}
        """.trimIndent()
    }
}
